src/data_import.py

```diff
@@ -3,39 +3,38 @@
 import backup
 import versions
 from docker_helpers import docker_cp
-from service import service
 from teamcity import save_artifact
 
 
-def do(settings):
-    source = settings["initial_data_source"]
+def do(service):
+    source = service.settings["initial_data_source"]
     print("Running initial data import with mode: {}".format(source))
     if source == "minimal":
         print("- Nothing to do (migrations will insert minimal data)")
     elif source == "test_data":
         local_path = get_artifact("montagu_api_generate_test_data", "test-data.sql", commit_hash=versions.api)
-        import_sql(local_path)
+        import_sql(db, local_path)
```

Comment:
Where does this `db` variable come from?

Reply:
Nice catch - this is one of the untested bits.  That should be `service.db`